### ISSUE TITLE
Session problem on two servers on one IP and two ports

### ISSUE THREAD

**Issue by fakhamatia** [phpmyadmin/phpmyadmin#16459]
### Describe the bug

I have 2 servers behind a modem
LAN HTTP port 80
WAN HTTP port server 1 = 8080
WAN HTTP port server 2 = 8081
When I use PHPMyAdmin on server 1, server 2 logout from PHPMyAdmin

### Expected behavior

Login to PHPMyAdmin on two servers same time

### Screenshots

![Peek 2020-11-05 17-11](https://user-images.githubusercontent.com/20030910/98249236-45fc2600-1f8b-11eb-89f0-96fce84eacc2.gif)

### Server configuration

- Operating system: Ubuntu 18.04
- Web server: Apapche
- Database version: MySQL 5.7
- PHP version: 7.2
- phpMyAdmin version: 7.2.24-0ubuntu0.18.04.6

### Client configuration

- Browser: Google Chrome 86.0.4240.183 
- Operating system: Ubuntu 20.04

**Comment by williamdes** [comment]
Hi @fakhamatia 
This is not a valid version`phpMyAdmin version: 7.2.24-0ubuntu0.18.04.6`
Please have a look on the homepage and let me know the version you are using.

**Comment by fakhamatia** [comment]
> Hi @fakhamatia
> This is not a valid version`phpMyAdmin version: 7.2.24-0ubuntu0.18.04.6`
> Please have a look on the homepage and let me know the version you are using.

```bash
:~$ sudo apt install phpmyadmin 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
phpmyadmin is already the newest version (4:4.9.7+dfsg1-1bionic1).
```
![ksnip_20201105-173638](https://user-images.githubusercontent.com/20030910/98251101-89579400-1f8d-11eb-923d-6788857e352d.png)

**Comment by fakhamatia** [comment]
> Hi @fakhamatia
> This is not a valid version`phpMyAdmin version: 7.2.24-0ubuntu0.18.04.6`
> Please have a look on the homepage and let me know the version you are using.

Sorry 
4.9.7deb1bionic1

**Comment by williamdes** [comment]
Thank you ;)
You are using the bionic PPA :100: 
How did you setup your phpMyAdmin instances ?

**Comment by fakhamatia** [comment]
When installing phpMyAdmin from the terminal, wizard setup, but not 4.9.7, I update it today, I install from PPA in 2020/06

**Comment by williamdes** [comment]
> When installing phpMyAdmin from the terminal, wizard setup, but not 4.9.7, I update it today, I install from PPA in 2020/06

So to sum up you only want to "Login to PHPMyAdmin on two servers same time" 

The error message did mistake me in the comprehension of this issue

**Comment by fakhamatia** [comment]
> So to sum up you only want to "Login to PHPMyAdmin on two servers same time"

Yes, Login to PHPMyAdmin on two servers same time with one IP but two (or more) different port

**Comment by williamdes** [comment]
> > So to sum up you only want to "Login to PHPMyAdmin on two servers same time"
> 
> Yes, Login to PHPMyAdmin on two servers same time with one IP but two (or more) different port

As an alternative you can use hostnames from `/etc/hosts` for now

**Comment by fakhamatia** [comment]
Do you have any ideas on how to solve this problem?
I have this problem on my site on different ports
I do not use sessions, only the browser cookie and it does not include the port

**Comment by williamdes** [comment]
> Do you have any ideas on how to solve this problem?
> I have this problem on my php site on different ports
> I do not use sessions, only the browser cookie and it does not include the port

I am not sure how we can fix this :thinking: 
@ibennetch or @mauriciofauth do you have an idea ?

**Comment by williamdes** [comment]
This issue is a bit strange
@fakhamatia could you upgrade to 5.1 and let us know if this is still accurate ?

**Comment by fakhamatia** [comment]
> This issue is a bit strange
> @fakhamatia could you upgrade to 5.1 and let us know if this is still accurate ?

How to upgrade to 5.1 with repository and PPA? in Ubuntu 18.04

**Comment by williamdes** [comment]
You might need this: [Wiki for install methods on Ubuntu/Debian](https://github.com/phpmyadmin/phpmyadmin/wiki/DebianUbuntu#related-issues-and-comments-on-install-methods)

**Comment by fakhamatia** [comment]
> This issue is a bit strange
> @fakhamatia could you upgrade to 5.1 and let us know if this is still accurate ?

I upgraded to 5.1.1 but nothing changed.
When I login to server1, server 2 is logout.
Even if I refresh login page in server1, server2 is logout.

**Comment by williamdes** [comment]
Something could be wrong in the scope of the session cookie, I will try to find out
Thanks

**Comment by fakhamatia** [comment]
I fix this problem in my panel.
Adding `$_SERVER['SERVER_PORT']` to end of cookie name, like:
```php
$_COOKIE["username_" . $_SERVER['SERVER_PORT']]
```

**Comment by williamdes** [comment]
I qualified this as a bug so we can have a better look at how it can be fixed
The port is definitely something we should consider
